### ISSUE TITLE
Project-1 Proposal

### ISSUE THREAD

**Issue by sashasabov** [sashasabov/Project-1#1]
# Project 1: Sight Word Pro!
## Project Description
This project has a form of educational game for kindergarten age kids to help them learn sight words and have fun. This game has three levels:

1. User has to catch letters of the chosen sight word. Word is displayed.
2. User sets timer and then uses keyboard to type the word within the time limit.  Word is displayed.
3. User has to catch the sight word on the moving object. Word is not displayed. 

I'm hoping to be able to add features for parents like uploading sight words for the game and favorite stickers of the child as winning price. This way parent will be a part of the game, also helping to read and pronounce selected word.

## Wire Frames

- Selecting sight word and starting the game. 

![Screen Shot 2022-04-27 at 2 15 42 PM (2)](https://user-images.githubusercontent.com/101619380/165593233-401d84e1-e9cc-4f13-8569-c738e09ee424.png)

- Level1: Catch all letters of the displayed word in consecutive  order. Then, move on to Level 2.

<img width="393" alt="Screen Shot 2022-04-27 at 2 43 17 PM" src="https://user-images.githubusercontent.com/101619380/165597711-e22b1624-4357-497d-a6a4-b887db6b8cd7.png">

- Level 2: Set timer and type displayed word. Then, move on to Level 3.

<img width="449" alt="Screen Shot 2022-04-27 at 2 47 39 PM" src="https://user-images.githubusercontent.com/101619380/165598063-94087437-5cad-4146-a4e4-f6e0b8651777.png">

- Level 3: Find and catch the sight word. Word will not be displayed this time. 

<img width="379" alt="Screen Shot 2022-04-27 at 2 50 26 PM" src="https://user-images.githubusercontent.com/101619380/165598611-97fbe78f-4515-4e69-a3ca-7fafb0a12532.png">

### MVP Goals

- As a player, I would like to start the game once words are selected.
- As a player, I would like to see indication when right letter is selected.
- As a player, I would like to change the level once first is complete.
- As a player, I would like to be able to set the timer for Level 2.
- As a player, I would like to start over from the same Level if I don't complete the task within set time.
- As a player, I would like to see indication when I click the right button on keyboard.
- As a player, I would like to be able to move on to Level 3 once second is completed.
- As a player, I would like to have option on a Hint to look at the word that I need to catch in case I forget it. I'm a kid after all!

## Stretch Goals

- As a player, I would like to be able to select background of my game.
- As a player, I would like to select sticker of a winner at the end of the game.
- As a player, I would like to keep track of progress of my game. 
- As a player, I would like to be able to share my progress with others. 
- As a player, I would like to be able to change the speed of moving objects.
- As a player, I would like to add music to the game. 

© 2022 Sasha Sabov.
 Inspired by Andrew Sabov.
